schemas/JSON/settings/settings.schema.0.2.json

```diff
@@ -184,6 +184,11 @@
           "type": "integer",
           "default": 3,
           "minimum": 1
+        },
+        "archiveExtractionMethod": {
+          "description": "Controls the use of tar.exe for extracting packages.",
+          "type": "boolean",
```

Comment:
Not a boolean

Reply:
Apologize Kabel. I don't know why I made a boolean for it. I guess I was reading too many settings. Do you mind re-reviewing? I made an enum now.